vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/DatacenterMO.java

```diff
@@ -82,6 +82,49 @@ public VirtualMachineMO findVm(String vmName) throws Exception {
         List<ObjectContent> ocs = getVmProperties(new String[] {"name", instanceNameCustomField});
         return HypervisorHostHelper.findVmFromObjectContent(_context, ocs.toArray(new ObjectContent[0]), vmName, instanceNameCustomField);
     }
+    public VirtualMachineMO findVm2(String vmName) throws Exception {
```

Comment:
Can we rename the `findVm` method to something similar to `findVmThroughPropertiesEx` and rename this `findVm2` method to `findVm`? This way we can redirect all the existing usages of the method to it. Then we could invoke `findVmThroughPropertiesEx` to the listUnmanagedInstances execution